operators/aqua/2022.4.1/manifests/aqua-operator.v2022.4.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,566 @@
+      mediatype: image/svg+xml
+  miniKubeVersion: 1.11.0
```

Comment:
You are using v1 CRD API, which was introduced with k8s 1.16
So, the minimal Kubernetes version where it can work well is 1.16.0 and not 1.11.0

c/c @yossig-aquasec

Reply:
thanks, done